packages/kit/types/index.d.ts

```diff
@@ -174,58 +176,256 @@ export interface Cookies {
 }
 
 export interface KitConfig {
+	/**
+	 * Your [adapter](https://kit.svelte.dev/docs/adapters) is run when executing `vite build`. It determines how the output is converted for different platforms.
+	 * @default undefined
+	 */
 	adapter?: Adapter;
+	/**
+	 * An object containing zero or more aliases used to replace values in `import` statements. These aliases are automatically passed to Vite and TypeScript.
+	 *
+	 * ```js
+	 * /// file: svelte.config.js
+	 * /** @type {import('@sveltejs/kit').Config} *\/
```

Comment:
This makes the whole code snippet unreadable in VS Code. I think it would be better to either have something like 
```js
/// type: import('@sveltejs/kit').Config
```
above things like this which get transformed to `/** @type ... */`, or leave out jsdoc comments inside jsdoc completely (I already added some code [`read_d_ts_file`] so that it doesn't fail in this case)

Reply:
good catch — done